runtime/src/main/java/org/corfudb/runtime/clients/LogUnitClient.java

```diff
@@ -33,6 +35,10 @@
     @Getter
     IClientRouter router;
 
+    public String getHost() { return "test-host"; }
```

Comment:
Yes, `getHost()` should be returning the value of `router.getHost()`, sorry, will fix.

About your host:ip prefix comment, I don't understand, sorry.  There is no single value for the metric name.  (This applies to your comment 4 days ago in NettyClientRouter also).

getHost() and getPort() are used to embed the Corfu server endpoint name into the metric so that each runtime client can keep separate synchronous op response time histograms (and counters for other events like # of disconnections) on a per-server basis.